loop/storage/redis.js

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 "use strict";
-var redis = require("redis");
+var redis = require("./redis_client");
```

Comment:
Oh by default it doesn't shutdown the multi feature. It does only if `sharding === true`.